styles/Vocab/Stakater/accept.txt

```diff
@@ -242,3 +242,5 @@ xfs
 yaml
 YAML
 Zerotier
+NDA[s]?
+Signup
```

Comment:
signup can be lowercase too

Reply:
you can also just write "sign up" which is accepted, so no need to add this

Reply:
On websites we see Signup don't we?

Reply:
which website?